### ISSUE TITLE
Compilation

### ISSUE THREAD

**Issue by ghost** [limneos/MobileMiner#20]
@limneos 
Could you please give us step by step instructions on how you compiled the minerd.a file. Many of us would like to compile it ourselves but are not able to because of various errors.

**Comment by nathjyo** [comment]
is the precompiled minerd.a supplied no good ?

**Comment by ghost** [comment]
No, but I would like to know how to compile my own version but @limneos won't respond.
